test/conformance/source/environment.cpp

```diff
@@ -199,6 +201,14 @@ DevicesEnvironment::DevicesEnvironment(int argc, char **argv)
         error = "Could not find any devices associated with the platform";
         return;
     }
+    // Get an environment variable (NUMBER_OF_DEVICES) to run tests on the appropriate number of devices
+    auto devices_num = ur_getenv("NUMBER_OF_DEVICES");
```

Comment:
hm, can't we just take arguments from the cli like all the rest cts configuration options?

Reply:
I think the option with the env variable is much easier and better in this case. It will be hard to pass this as an argument in the CTS configuration because of the complexity of the tests.  I would leave it that way.

Reply:
I've changed it